kmp/reporters_test.go

```diff
@@ -217,7 +217,9 @@ func TestImmutableReporter(t *testing.T) {
 		x: testStruct{
 			StringField: "foo",
 		},
-		expectErr: true,
+		want: `root:
+	-: "{A: StringField:foo IntField:0 StructField:{ChildString: ChildInt:0} Omit: Ignore: Dash: MultiComma:}"
+`,
 	}, {
 		name: "Single character field name",
```

Comment:
Can you add a test case for a map with a missing key since that is where you are seeing the bug?

Reply:
done